### ISSUE TITLE
training question

### ISSUE THREAD

**Issue by jimmy9704** [W-zx-Y/DANNet#10]
Each time i trained DANNet the performance is derived differently. 
Such as in Dark zurich val dataset miou= 35.5 ~ 36.5.
Is this because of gan's learning difficulties?

**Comment by WANGLEKV** [comment]
and  in the testing step, the trained model (PSP/deeplab) can only show an IOU of 36.76……

**Comment by W-zx-Y** [comment]
Hi @jimmy9704,

> Each time i trained DANNet the performance is derived differently. Such as in Dark zurich val dataset miou= 35.5 ~ 36.5. Is this because of gan's learning difficulties?

You are right that the training of GAN is not very stable. As shown in Figure 7, 36.76 is achieved by adjusting the mean and std using the validation set. And this model with the picked mean and std are applied to the testing set.

**Comment by W-zx-Y** [comment]
> and in the testing step, the trained model (PSP/deeplab) can only show an IOU of 36.76……

Hi @WANGLEKV,

It is possible to get mIoU around 36 on the validation set. On the test set, the mIoU is usually around 45. Is 36.76 achieved on the testing set? What's the performance on the val set?
